### DISCUSSION TITLE
Really broken on Linux?

### DISCUSSION THREAD

**Discussion by hello-world-dot-c** [iterate-ch/cyberduck#15357] [Q&A]
Just starting to use Cyberduck on a headless machine and can't get past OAuth for accessing the Dropbox storage we use. Does [this issue](https://github.com/iterate-ch/cyberduck/issues/13587) mean that this tool has been defunct for Linux for months now, and is expected to stay that way? Or am I misinterpreting it?

**Comment by dkocher** [comment]
At least for Dropbox we should be able to create an alternate connection profile using a redirect URI that allows to copy & paste the authorization code.

**Comment by hello-world-dot-c** [comment]
I understand it has been added to the [milestones for 9.0](https://github.com/iterate-ch/cyberduck/milestone/182) so never mind.

**Comment by dkocher** [comment]
For Dropbox you could register your own client application [^1] and create a matching connection profile [^2] for the OAuth Client Id registration. I will attach a connection profile that is usable with Cyberduck CLI in #13587.

[^1]: https://www.dropbox.com/developers/apps
[^2]: https://docs.cyberduck.io/protocols/profiles/

**Comment by hello-world-dot-c** [comment]
Thank you, how am I supposed to use this? I tried

`duck --profile ~/Dropbox\ \(CLI\).cyberduckprofile --list dropbox:/`

which returns

```
Reading metadata of /…
No login credentials could be found in the Keychain.
Try Again (y) or Cancel (n):  (y/n): n

Unknown. Unknown.
```

and

`duck --username ******* --profile ~/Dropbox\ \(CLI\).cyberduckprofile --list dropbox:/`

returns

```
Resolving…
Unknown. Cannot invoke "ch.cyberduck.core.PathAttributes.getRegion()" because the return value of "ch.cyberduck.core.Path.attributes()" is null.
```

Same with giving --password. I have created a "scoped app" in Dropbox with app key, app secret and generated access token but I don't get to enter this anywhere.
